src/core/scene/a-scene.js

```diff
@@ -270,13 +271,15 @@ class AScene extends AEntity {
    * @param {bool?} useAR - if true, try immersive-ar mode
    * @returns {Promise}
    */
-  enterVR (useAR) {
+  enterVR (useAR, useOfferSession) {
     var self = this;
     var vrDisplay;
     var vrManager = self.renderer.xr;
     var xrInit;
 
     // Don't enter VR if already in VR.
+    if (useOfferSession && (navigator.xr === undefined || navigator.xr.offerSession === undefined)) { return Promise.resolve('OfferSession is not supported.'); }
```

Comment:
can shorten to `if (useOfferSession && (!navigator.xr || !navigator.xr.offerSession))`